### ISSUE TITLE
Add support for "isolate" value in "unicode-bidi"

### ISSUE THREAD

**Issue by Krinkle** [CSSLint/csslint#436]
The following should afaik not generate any warnings:

``` css
.mw-example {
    unicode-bidi: -moz-isolate;
    unicode-bidi: -webkit-isolate;
    unicode-bidi: isolate;
}
```

.. as it does right now:

> - Expected (normal | embed | bidi-override | inherit) but found '-moz-isolate'.
> - Expected (normal | embed | bidi-override | inherit) but found '-webkit-isolate'. 
> - Expected (normal | embed | bidi-override | inherit) but found 'isolate'.
> 
> Properties should be known (listed in CSS3 specification) or be a vendor-prefixed property. (`known-properties`)

See also:
- [CSS/unicode-bidi](https://developer.mozilla.org/en-US/docs/Web/CSS/unicode-bidi) on MDN

**Comment by Krinkle** [comment]
Switched to <https://stylelint.io/>.
